### PR TITLE
rfop_ruo_api

### PR DESCRIPTION
**Pull request by Vital-jan**
API structure for rfop and ruo

### CODE REVIEW

File: ratu/serializers.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from rest_framework import serializers
+
+class RfopSerializer(serializers.Serializer):
+    state = serializers.CharField(max_length=120)
+    kved = serializers.CharField()
```
Comment (Iva-khar) [inline ratu/serializers.py:5]: same ^

File: ratu/serializers.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    fullname = serializers.CharField()
+    address = serializers.CharField()
+
+class RuoSerializer(serializers.Serializer):
+    state = serializers.CharField(max_length=120)
+    kved = serializers.CharField()
```
Comment (Iva-khar) [inline ratu/serializers.py:11]: same^

File: data_converter/urls.py [hunk 3]
```diff
@@ -13,17 +13,11 @@
...
-
-# urlpatterns = [
-#     path('admin/', admin.site.urls),
-# ]
 
 from django.contrib import admin
 from django.urls import include, path
 
 urlpatterns = [
-    path('ratu/', include('ratu.urls')),
```
[the remark this answers is not in the document]
Comment (Vital-jan) [reply data_converter/urls.py:27]: Yes. This line not need at this module.

File: ratu/serializers.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+from rest_framework import serializers
+
+class RfopSerializer(serializers.Serializer):
+    state = serializers.CharField(max_length=120)
```
[the remark this answers is not in the document]
Comment (Vital-jan) [reply ratu/serializers.py:4]: This code uses to view object. It`s correct.

File: ratu/urls.py [hunk 5]
```diff
@@ -1,7 +1,17 @@
 from django.urls import path
+from .views import RfopView
+from .views import RuoView
```
[the remark this answers is not in the document]
Comment (Vital-jan) [reply ratu/urls.py:3]: fixed.

**Pull request merged**